extensions/mssql/src/objectExplorerNodeProvider/objectExplorerNodeProvider.ts

```diff
@@ -299,31 +289,34 @@ class RootNode extends TreeNode {
 }
 
 class DataServicesNode extends TreeNode {
-	private children: TreeNode[];
-	constructor(private _session: Session, private context: TreeDataContext, private nodePath: string) {
+	private _children: TreeNode[];
+	private _sqlClusterSession: SqlClusterSession;
+	private _treeDataContext: TreeDataContext;
+	private _nodePath: string;
+	constructor(sqlClusterSession: SqlClusterSession, treeDataContext: TreeDataContext, nodePath: string) {
 		super();
+		this._sqlClusterSession = sqlClusterSession;
+		this._treeDataContext = treeDataContext;
+		this._nodePath = nodePath;
 	}
 
-	public get session(): Session {
-		return this._session;
+	public get sqlClusterSession(): SqlClusterSession {
```

Comment:
As mentioned, just `session` should be fine and save space

Reply:
fixed.